lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -192,6 +192,16 @@ defmodule Bamboo.SMTPAdapter do
   end
 
   defp add_attachment_header(body, attachment) do
+    case attachment.content_id do
```

Comment:
Looking at [`Bamboo.Attachment`](https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/attachment.ex), I don't understand where the key `content_id` is defined?